### PR TITLE
kommander: bump kcl and clean up chart

### PR DESCRIPTION
**Pull request by makkes**
Since a lot of the resources in this chart have been moved to the kcl chart itself, I removed them from this chart.

### CODE REVIEW

**Comment by jimmidyson** [comment]
I think there will be some stuff in values to clean up too?

**Comment by makkes** [comment]
> ?

Fixed.

**Comment by jimmidyson** [comment]
https://teamcity.mesosphere.io/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=2534551&_focus=442

> [ERROR] templates/: render error in "kommander/templates/hooks-kubeaddons.yaml": template: kommander/templates/hooks-kubeaddons.yaml:1:14: executing "kommander/templates/hooks-kubeaddons.yaml" at <.Values.kubefed.enab...>: can't evaluate field enabled in type interface {}

File: stable/kommander/requirements.yaml [hunk 1]
```diff
@@ -1,14 +1,10 @@
 dependencies:
-- name: kubefed
```
Comment (alejandroEsc) [inline stable/kommander/requirements.yaml]: So kubefed is being removed? Where is this applied now? and wouldnt some crds need to be removed now? what are the assumptions here?

Comment (makkes) [reply stable/kommander/requirements.yaml]: That stuff has been moved to the kcl repohttps://github.com/mesosphere/kommander-cluster-lifecycle/pull/313

**Pull request merged**